src/Common/src/Interop/Unix/System.Native/Interop.SchedGetSetAffinity.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal unsafe struct CpuSetBits
```

Comment:
One more thought.  Since the Bits field is never used and is here purely to impact the size of the struct, you could just add a StructLayout attribute to the type to set its size to 128.  Then the type wouldn't need to be unsafe.  But if you'd prefer to stick with what you have, that's fine, though you might chaynge the visibility of Bits to be private.